platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/DataFileCollectionTest.java

```diff
@@ -195,43 +139,27 @@ void createDataFileCollection(FilesTestType testType) throws Exception {
                 // store in file
                 storedOffsets.put(i, fileCollection.storeDataItem(dataValue));
             }
-            fileCollection.endWriting(0, count + 100).setFileCompleted();
+            final DataFileReader<long[]> newFile = fileCollection.endWriting(0, count + 100);
+            newFile.setFileCompleted();
             assertEquals(new KeyRange(0, count + 100), fileCollection.getValidKeyRange(), "Range should be this");
+            assertEquals(Files.size(newFile.getPath()), newFile.getSize());
             count += 100;
         }
         // check 10 files were created
         assertEquals(10, Files.list(tempFileDir.resolve(testType.name())).count(), "unexpected file count");
     }
 
-    @Order(2)
-    @Test
```

Comment:
Why is this test no longer relevant?

Reply:
It can probably be kept for JDB files, but not for PBJ. The test uses default DataFileCollection implementation, which may create either JDB files or PBJ files, based on config. File sizes will be different. To avoid test changes, when we change the default value for "usePbj" config, file size checks are removed. They aren't very helpful anyway, from my perspective. Also, PBJ files don't have footers checked in this test, one more reason to drop.